packages/blockchain-wallet-v4-frontend/src/scenes/Interest/IntroCard/index.tsx

```diff
@@ -43,6 +46,65 @@ const IneligibleBanner = styled.div`
 class IntroCard extends PureComponent<
   ParentStateType & Props & SuccessStateType
 > {
+  renderAdditionalInfo = () => {
+    const { analyticsActions, interestRateArray } = this.props
+    const highestRate = interestRateArray.reduce((a, b) => Math.max(a, b))
+    return (
+      <BoxStyledAdditional>
+        <ContentWrapper>
+          <IconWrapper>
+            <Icon color='blue600' name='percentage' size='32px' />
+          </IconWrapper>
+          <Text
+            size='20px'
+            color='grey800'
+            weight={600}
+            style={{ marginTop: '16px' }}
+          >
+            <FormattedMessage
+              id='scenes.interest.additional_info_required'
+              defaultMessage='Additional Info Required'
+            />
+          </Text>
+          <Text
+            size='14px'
+            color='grey600'
+            weight={500}
+            style={{ marginTop: '4px', lineHeight: 1.5 }}
+          >
+            <FormattedMessage
+              id='scenes.interest.additional_info_required_description'
+              defaultMessage='Please supply the additional information required to avoid delays on withdrawal.'
+              values={{ highestRate }}
+            />
+          </Text>
+          <Link
+            href='https://share.hsforms.com/1DS4i94fURdutr8OXYOxfrg2qt44'
```

Comment:
maybe make this a config or a constant somewhere since it's used in a couple places?

Reply:
yeah, will move to const so we can reuse, we might need some sort of global config location for such